template/{{.input_root_dir}}/docs/ml-pull-request.md.tmpl

```diff
@@ -78,14 +78,17 @@ After merging your pull request, subsequent runs of the model training and batch
 jobs in staging and production will automatically use your updated ML code.
 {{- end }}
 
-You can track the state of the ML pipelines for the current project from the MLflow registered model UI. Links:
+You can track the state of the ML pipelines for the current project from the MLflow registered model UI. 
+{{ if or (eq .input_setup_cicd_and_project `CICD_and_Project`) (eq .input_cicd_platform `Project_Only`) }}
```

Comment:
We don't need this `or`. This file will never be generated for the `Project_Only` case